app/src/main/res/values-it/strings.xml

```diff
@@ -315,8 +315,8 @@
     <string name="tx_account">Account #</string>
 
     <string name="send_sweepall">Manda tutti i fondi confermati in questo account!</string>
-    <string name="tx_subaddress">Subaddress #%1$d</string>
-    <string name="generate_address_label_sub">Subaddress pubblico #%1$d</string>
+    <string name="tx_subaddress">Sottoindirizzo #%1$d</string>
```

Comment:
I would leave 'subadress' as it is for now. 'sottoindirizzo' could be confusing. I think this is a discussion for monero-ecosystem/monero-translations

Reply:
I reverted the latest commit in order to take the discussion to  monero-ecosystem/monero-translations.
The review can continue.